spec/lib/roadie/document_spec.rb

```diff
@@ -1,4 +1,6 @@
 # encoding: UTF-8
```

Comment:
We should remove all of these magic comments too; I don't think they do anything after Ruby 2.2.

I don't mind doing it after merging this in case you are not interested in doing it. Just let me know. :smile:

Reply:
Not a problem, took care of it